app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/commontask/CommonTaskScreen.kt

```diff
@@ -240,7 +241,10 @@ fun CommonTaskScreen(
             navigateBack = navController::popBackStack,
             navigateToCreateTask = { navController.navigateToCreateTaskScreen(CommonTaskType.Task, commonTaskId) },
             navigateToTask = navController::navigateToTaskScreen
-        )
+        ),
+        navigateToProfile = { userId ->
+            navController.navigate("${Routes.profile}/$userId")
```

Comment:
It's better to create separate function here for this navigation action:
https://github.com/EugeneTheDev/TaigaMobile/blob/master/app/src/main/kotlin/io/eugenethedev/taigamobile/ui/utils/NavigationUtils.kt